TcUnit-Runner/Program.cs

```diff
@@ -119,14 +122,16 @@ static void Main(string[] args)
             try
             {
                 vsInstance = new VisualStudioInstance(@VisualStudioSolutionFilePath, tcVersion);
-                vsInstance.Load();
+                var tcPinnedVersion = XmlUtilities.GetPinnedVersionXML(TwinCATProjectFilePath);
+                log.Info("Version is pinned: " + tcPinnedVersion);
```

Comment:
It seems the TcPinnedVersion is a boolean, and not the actual version?

Reply:
Yes it is a boolean. 
"TcVersionIsPinned " would that be a better name

Reply:
Or IsTcVersionPinned ... :-)